### ISSUE TITLE
VPAT: Sitewide

### ISSUE THREAD

**Issue by dlowenberg** [CDL-Dryad/dryad-product-roadmap#1326]
Spreadsheet: https://docs.google.com/spreadsheets/d/14PwHqY7rnlEAemad-Dsn0VNADFODudTZ9q-oVfu6Ffc/edit#gid=545873482

Make changes to match spec. Change all ""link-type"" (blue text, no fill or border, underline or underline on hover) to secondary buttons (black on powder blue). This includes action on My Datasets and remove buttons on describe and upload pages. Or, ideally, conduct an overall button review to rationalize presentation and semantics.

Specs:
https://drive.google.com/file/d/1uyYRX6gKfs-ga6Lt04qh4776K_ibh6QS/view?usp=sharing

https://3r06p1.axshare.com/#id=c59f37&p=my-datasets_alt-buttons&c=1

https://3r06p1.axshare.com/#id=h9gi31&p=upload_data_v7_static_popup-report_alt-buttons&c=1

**Comment by sfisher** [comment]
Using role="button" and similar for the places these are suspect.  While displaying full-size buttons might be clearer, it would throw the layout off and be different from the design and we can discuss if these aria changes are sufficient.
